src/Magnum/Shaders/Generic.h

```diff
@@ -150,13 +150,13 @@ both bitangents and object ID for instancing, \n
 <tr>
 <td>10</td>
 <td colspan="2">
-* *Reserved* --- 2nd vertex weights
+@ref Weights
```

Comment:
This is still wrong -- either add `SecondaryWeights`, `SecondaryJointIds` or revert this and the below change.

Reply:
Will make it SecondaryWeights 👍